sources/java2/src/main/java/com/kaltura/client/utils/request/ExecutedRequest.java

```diff
@@ -10,6 +11,7 @@
     String response = null;
     boolean isSuccess = false;
     ErrorElement error = null;
+    Headers headers = null;
```

Comment:
Please use Map<String, String> instead of specific library class.

Reply:
done